integration/qbft/scenarios/regular_aggregator.go

```diff
@@ -82,14 +69,14 @@ func regularInstanceValidator(consensusData *spectypes.ConsensusData, operatorID
 			Data: encodedConsensusData,
 		}).Encode()
 		if err != nil {
-			return fmt.Errorf("encode prepare data: %w", err)
+			panic(err)
```

Comment:
https://github.com/bloxapp/ssv/pull/793#discussion_r1067178852 (you can just pull from stage)

Reply:
it overwrites on future brunches